pyk/src/pyk/kcfg.py

```diff
@@ -58,26 +58,29 @@ def to_dict(self) -> Dict[str, Any]:
 
         def to_rule(self, claim=False, priority=50) -> KRuleLike:
             sentence_id = f'BASIC-BLOCK-{self.source.id}-TO-{self.target.id}'
-            init_term = mlAnd([self.source.term, self.condition])
-            final_term = self.target.term
-            return buildRule(sentence_id, init_term, final_term, claim=claim, priority=priority)
+            init_term = mlAnd([self.source.cterm.term, self.condition])
+            final_term = self.target.cterm.term
+            rule, _ = buildRule(sentence_id, init_term, final_term, claim=claim, priority=priority)
+            return rule
```

Comment:
`self.source.cterm.term` returns the term _and_ its constraints? I just want to confirm that we're calling `buildRule` with the appropriate arguments.